products/shopware.md

```diff
@@ -0,0 +1,184 @@
+    extendedSupport: false
+    latest: "5.2.27"
+    latestReleaseDate: 2017-07-17
+    link: false
+
+-   releaseCycle: "5.1"
+    releaseDate: 2015-10-21
+    support: false
+    eol: true
+    extendedSupport: false
+    latest: "5.1.6"
+    latestReleaseDate: 2016-05-23
+    link: false
+
+-   releaseCycle: "5.0"
+    releaseDate: 2015-04-27
+    support: false
+    eol: true
+    extendedSupport: false
+    latest: "5.0.4"
+    latestReleaseDate: 2015-09-16
+    link: false
+
+---
+
+> [Shopware](https://shopware.com) is an [MIT licensed](https://github.com/shopware/platform/blob/trunk/LICENSE)
+> e-commerce platform written in PHP. The project is made by the same-named vendor, Shopware AG.
+
+Shopware follows [semantic versioning](https://semver.org/).
+
+## Feature and bug fix releases
+
+In Shopware 6, bug and security fixes might appear in every patch release, whereas features, bug, and security fixes might appear in minor releases as well as in major releases. Additionally, in major releases, there will appear backward compatibility breaks.
+
+## Security
+
+Shopware is an Open Source software vendor and takes security issues very seriously. Here, you will find information on how to report security-related issues to us and how we process such issues.
+
+If you discover a security issue in one of our products or services, please get in touch with us immediately. Our policy is to limit public knowledge about a security issue until we provide a fix for it.
+
+We kindly ask you to inform us first and keep the vulnerability confidential for the public, as this will compromise existing businesses.
+
+* Information on how to report security issues in Shopware: [shopware.com/security.txt](https://www.shopware.com/security.txt) 
+* If you are unable to update to the latest Shopware version, you are able to protect your shop from vulnerabilities using the [Shopware Security Plugin](https://store.shopware.com/en/swag575294366635f/shopware-security-plugin.html).  To protect yourself from possible security vulnerabilities, download and activate the plugin in the administration panel of your shop. It is important to always keep this plugin updated to the latest version. 
+
+## Extended support for Shopware 5
+
+* All repositories of Shopware 5 are available on GitHub: [github.com/shopware5/](https://github.com/shopware5/); the vendor _shopware AG_ is transferring the project to the community and is ending all support effective August 2024.
+* If you have to use Shopware 5 in a project after the official support has ended,  [safefive](https://safefive.de/en/home/) is offering security and compatibility updates as a commercial service.
+
+## Release notes
+
+* [github.com/shopware/shopware/blob/trunk/CHANGELOG.md](https://github.com/shopware/shopware/blob/trunk/CHANGELOG.md)
+
+## System requirements
+
+* [docs.shopware.com/en/shopware-6-en/first-steps/system-requirements](https://docs.shopware.com/en/shopware-6-en/first-steps/system-requirements)
```

Comment:
Readers of this page already know what Shopware is, so these aren't needed. Plus, we have better links to release notes in the table already.
```suggestion
```